web-common/tests/utils/assertUrlParams.ts

```diff
@@ -0,0 +1,7 @@
+import { expect, type Page } from "@playwright/test";
+
+export function assertUrlParams(page: Page, expectedParams: string) {
+  expect(new URL(page.url()).searchParams.toString()).toMatch(
+    new URL("http://localhost?" + expectedParams).searchParams.toString(),
```

Comment:
I see `http://localhost?`, and it looks like a port is missing. But anyways, to test against `expectedParams`, what about just creating a `URLSearchParams` object rather than a full `URL` object?